server/src/jira/jira.controller.ts

```diff
@@ -0,0 +1,46 @@
+import { Controller, Get, Res, UseGuards } from "@nestjs/common";
+import { Response } from "express";
+import { BypassHeadersGuard } from "src/decorators/bypass-headers-guard.decorator";
+import { JiraAuthGuard } from "./jira.guard";
+import { JiraService } from "./jira.service";
+import { SessionTokenGuard } from "./token.guard";
+import { ConfigService } from "src/config/config.service";
```

Comment:
Use relative import. Note also `BypassHeadersGuard` above. Too bad that linting does not appear to catch these... :(

I recommend setting `javascript.preferences.importModuleSpecifier` and `typescript.preferences.importModuleSpecifier` to "relative" in VS Code settings to avoid it mixing different import styles. Also, setting `javascript.updateImportsOnFileMove.enabled` and `typescript.updateImportsOnFileMove.enabled` to "always" would result in cleaner diffs as then our editors would automatically order them the same way.